framework/include/base/MooseError.h

```diff
@@ -158,25 +158,31 @@
-  do                                                                                                        \
-  {                                                                                                         \
-    if (Moose::_deprecated_is_error)                                                                        \
-      mooseError("\n\nDeprecated code:\n" << msg << '\n');                                                  \
-    else                                                                                                    \
-      mooseDoOnce(                                                                                          \
-        _console                                                                                            \
-          << COLOR_YELLOW                                                                                   \
-          << "*** Warning, This code is deprecated, and likely to be removed in future library versions!\n" \
-          << msg << '\n'                                                                                    \
-          << __FILE__ << ", line " << __LINE__ << ", compiled "                                             \
-          << LIBMESH_DATE << " at " << LIBMESH_TIME << " ***\n";                                            \
-          if (libMesh::global_n_processors() == 1)                                                          \
-            print_trace(Moose::out);                                                                        \
-          else                                                                                              \
-            libMesh::write_traceout();                                                                      \
-          Moose::out << COLOR_DEFAULT << std::endl;                                                         \
-      );                                                                                                    \
+#define mooseDeprecated(msg)                                                                                  \
+  do                                                                                                          \
+  {                                                                                                           \
+    if (Moose::_deprecated_is_error)                                                                          \
+      mooseError("\n\nDeprecated code:\n" << msg << '\n');                                                    \
+    else                                                                                                      \
+      mooseDoOnce(                                                                                            \
+        {                                                                                                     \
```

Comment:
umm - no it's not. I added it when I had a compile error but it turns out my error was elsewhere. I can remove.